src/validation/obfl.rng

```diff
@@ -327,17 +334,12 @@
     <optional>
       <attribute name="keep">
         <choice>
-          <value>auto</value>
-          <value>page</value>
-          <value>sheet</value>
-          <value>volume</value>
+          <value>none</value>
+          <value>solid</value>
```

Comment:
Why did you change this?
It seems that the conversion of TABs to spaces in this file was not done correctly